lib/src/base-styles/_nx-colors.scss

```diff
@@ -488,6 +572,90 @@
     --nx-color-status-indicator-error-text: var(--nx-swatch-indigo-10);
     --nx-color-status-indicator-error-icon-background: var(--nx-swatch-red-40);
     --nx-color-status-indicator-error-icon-border: var(--nx-color-status-indicator-error-icon-background);
+
+    // NxSelectableColors
+    --nx-color-selectable-color-sky-dark: var(--nx-swatch-blue-80);
+    --nx-color-selectable-color-sky-light: var(--nx-swatch-blue-95);
+
+    --nx-color-selectable-color-purple-dark: var(--nx-swatch-purple-80);
+    --nx-color-selectable-color-purple-light: var(--nx-swatch-purple-95);
+
+    --nx-color-selectable-color-pink-dark: var(--nx-swatch-pink-80);
+    --nx-color-selectable-color-pink-light: var(--nx-swatch-pink-95);
+
+    --nx-color-selectable-color-blue-dark: var(--nx-swatch-teal-45);
+    --nx-color-selectable-color-blue-light: var(--nx-swatch-teal-90);
+
+    --nx-color-selectable-color-red-dark: var(--nx-swatch-red-70);
+    --nx-color-selectable-color-red-light: var(--nx-swatch-red-90);
+
+    --nx-color-selectable-color-turquoise-dark: var(--nx-swatch-turquoise-40);
+    --nx-color-selectable-color-turquoise-light: var(--nx-swatch-turquoise-90);
+
+    --nx-color-selectable-color-orange-dark: var(--nx-swatch-orange-65);
+    --nx-color-selectable-color-orange-light: var(--nx-swatch-orange-90);
+
+    --nx-color-selectable-color-yellow-dark: var(--nx-swatch-yellow-70);
+    --nx-color-selectable-color-yellow-light: var(--nx-swatch-yellow-90);
+
+    --nx-color-selectable-color-kiwi-dark: var(--nx-swatch-green-80);
+    --nx-color-selectable-color-kiwi-light: var(--nx-swatch-green-95);
+
+    --nx-color-selectable-color-indigo-dark: var(--nx-swatch-indigo-70);
+    --nx-color-selectable-color-indigo-light: var(--nx-swatch-indigo-95);
+
+    // NxTag
+    --nx-color-tag-text: var(--nx-swatch-indigo-10);
+
+    --nx-color-tag-sky-selected-hover-border: var(--nx-color-tag-sky-selected-hover-background);
```

Comment:
I'm not sure these variables are needed.  I think it would be ok to use the swatches directly in `_nx-tag-variables.scss`, including a `dark-mode` mixin block within each CSS block there.